matrix-meetings-widget/src/components/meetings/MemberSelectionDropdown/MemberSelectionDropdown.tsx

```diff
@@ -86,8 +74,26 @@ type MemberSelectionDropdownProps = {
   /** meeting id */
   meetingId?: string;
 
-  /** Show a button to select all members */
-  showSelectAll?: boolean;
+  /** Disables options filtering, makes sense when options are filtered already */
+  disableFilterOptions?: boolean;
+
+  /** If true, it shows the loadingText in place of suggestions. */
+  loading?: boolean;
+
+  /** Text to display when there are no options. */
+  noOptionsText?: ReactNode;
+
+  /** Text to display when in a loading state. */
+  loadingText?: ReactNode;
+};
+
+export type MemberSelection = {
+  state_key: string;
+
+  content: {
+    displayname?: string | null;
+    avatar_url?: string | null;
+  };
```

Comment:
I think we can make this even more independent from the state event.

How about:

```suggestion
  userId: string;
  displayName: string;
  avatarUrl?: string;
```

I would expect the caller to do the normalization (this includes the removal of the `null` value for the avatar_url).

In addition: lets move the type to the beginning of the file so the Props are nearer to the corresponding component.

Reply:
it is more independent now as suggested, normalization is applied